### PR TITLE
leaderboard project Hit the API part

### PR DESCRIPTION
**Pull request by haftamudesta**
This is the leaderboard project. The leaderboard website displays scores submitted by different players. It also allows you to submit your score, fetch scores from the API and render to the web.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,151 @@
...
+
+- **Can I modify it while practicing?**
+
+  - Yes, you can do it, but make sure that you know what are you doing, we never know when we can get stuck at some point just because we were playing with the code _-laughs-_
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (anagudelogu) [inline README.md]: - [ ] You did a good job working on your project documentation! 💯 However, please ensure all the links are working, this one is looking for a LICENSE file in your repo but you don't have one yet. Microverse suggests that you use the MIT one, you could set it following the steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

File: module/addItem.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          user: nameInput.value,
+          score: scoreInput.value,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      }
+    );
+    nameInput.value = '';
+    scoreInput.value = '';
```
Comment (anagudelogu) [inline module/addItem.js]: - [ ] **[OPTIONAL]:** You did a great job using the leaderboard API! 💯 However, I think these lines that serve as "resetting" the form can be better moved to the submit event listener, this way, your `sendData` function only has the responsibility to send the data to the API, and not also clear the input fields of the form! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @haftamudesta, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @haftamudesta , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by haftamudesta** [comment]
done

**Comment by haftamudesta** [comment]
done

**Pull request closed without merging**